itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/application/v3/basic/BuilderImageApplicationWizardHandlingTest.java

```diff
@@ -194,6 +191,12 @@ private void setDefaultValuesAndAssert(String defaultRepo, String defaultRef, St
 	private void validateGitRepoURL(String url) {
 		new LabeledText(OpenShiftLabel.TextLabels.GIT_REPO_URL).setText(url);
 		
+		new WaitUntil(new ControlIsEnabled(new NextButton()) {
```

Comment:
What do you want to achieve here?

Reply:
There is validation for correct URL, so I need to wait, until this validation finishes.